src/langcheck/metrics/eval_clients/_openai.py

```diff
@@ -62,8 +62,12 @@ def _call_api_with_exception_filter(model_input: dict[str, Any]) -> Any:
                 return e
 
         model_inputs = [
-            {"messages": [{"role": "user", "content": prompt}], **config}
-            for prompt in prompts
+            {
+                "messages": [{"role": "user", "content": prompt}],
+                "seed": i,
+                **config,
+            }
+            for i, prompt in enumerate(prompts)
```

Comment:
@liwii What do you think about just adding seed like this in `_call_api()`? I thought this would be the simplest but please let me know if you find any problems!

Reply:
Sounds good, I don't think we need to fix seeds if there are multiple requests.

Could you note that we call each prompt with different seeds since it is not a very straightforward behavior?